### ISSUE TITLE
Issue activating the package after installation

### ISSUE THREAD

**Issue by moxie** [yaroslavyaroslav/OpenAI-sublime-text#62]
Hi there,

I've followed the package installation instructions but am receiving the following error when Sublime is trying to load the package:

```
reloading plugin OpenAI completion.main
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/Lib/python38/sublime_plugin.py", line 325, in reload_plugin
    m = importlib.import_module(modulename)
  File "./python3.8/importlib/__init__.py", line 127, in import_module
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 868, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/joe/Library/Application Support/Sublime Text/Installed Packages/OpenAI completion.sublime-package/main.py", line 11, in <module>
  File "/Users/joe/Library/Application Support/Sublime Text/Installed Packages/OpenAI completion.sublime-package/plugins/buffer.py", line 6, in <module>
  File "/Applications/Sublime Text.app/Contents/MacOS/Lib/python38/sublime_types.py", line 5, in <module>
    from typing_extensions import TypeAlias
ModuleNotFoundError: No module named 'typing_extensions'
```

I've done some research but believe I understand that Sublime's python interpreter is private and not something I can add extensions to. Am I missing some kind of dependency here or does this look like an issue with my environment?

Thank you very much.

**Comment by yaroslavyaroslav** [comment]
Thanks for reaching out, what version of ST you are using?

**Comment by moxie** [comment]
> Thanks for reaching out, what version of ST you are using?

Thanks for the quick reply. I should have specified:

Sublime: Build 4180
MacOS: 14.4.1 (If helpful)

**Comment by yaroslavyaroslav** [comment]
Can you check in Finder is there something under this path: `~/Library/Application Support/Sublime Text/Lib/python38/typing_extensions.py `

**Comment by moxie** [comment]
I can confirm that in the `~/Library/Application Support/Sublime Text/Lib/python38` there is **no** `typing_extesions.py` - I have the folder but that file is not there.

**Comment by moxie** [comment]
After successfully reading between the lines here, I placed `typing_extensions.py` in the recommended folder and the package is working properly now.

**Comment by yaroslavyaroslav** [comment]
Yeah, it was actually missing dependency issue. Occasionally I had such in my env installed by some other package, so I faced not this issue on my side, while you had had less messy env on your side so it raise up. 

I've released 4.2.2 just yet that solves this issue (declares the missing dependency).
